src/NUnitFramework/framework/NUnitString.cs

```diff
@@ -43,6 +44,6 @@ public NUnitString(string? message)
         public static implicit operator NUnitString(string? message) => new(message);
```

Comment:
This is the one that causes the build errors.  
Do we need the nullable string in here?  (Was a comment on some other PR I think, and perhaps we need to tackle the Result class too? )

Reply:
Looked very quickly at it, seems like it can be non-null since you have fixed the nullability in the Result class, if I am not missing something.

Reply:
Somehow I missed this one. Fixed.